datajoint_utilities/dj_lazy_schema/lazy_schema.py

```diff
@@ -0,0 +1,133 @@
+import datajoint_utilities.typing as djt  # isort: skip
+
+import inspect
+from pathlib import Path
+from typing import Any, Mapping, NoReturn
```

Comment:
it's. usually better to import the entire module and then refer to the imported objects by the fully qualified name, e.g. `typing.Any`.